src/bundle/Resources/public/scss/fieldType/edit/_base-field.scss

```diff
@@ -44,6 +44,11 @@
     }
 }
 
+.ez-field-edit--disabled {
```

Comment:
This should share styling with `--nontranslatable` so above where there are styling for nontranslatable you should add just:
```scss
    &--nontranslatable,
    &--disabled {
```

Reply:
Done.